### PR TITLE
dotnet-init --> dotnet new

### PR DESCRIPTION
**Pull request by piotrpMSFT**
Fixes issue #374

### CODE REVIEW

**Comment by brthor** [comment]
Should change rootnamespace in Microsoft.DotNet.Tools.New.xproj

**Comment by brthor** [comment]
Also one more change for OSX pkg:
https://github.com/dotnet/cli/blob/master/packaging/osx/scripts/postinstall

@Sridhar-MS do changes need to be made for the MSI?

**Comment by Sridhar-MS** [comment]
No changes in MSI for this (no `symlinks` there).

File: src/Microsoft.DotNet.Tools.New/Program.cs [hunk 1]
```diff
@@ -72,13 +72,13 @@ public static int Main(string[] args)
...
-            app.Name = "dotnet init";
+            app.Name = "dotnet new";
             app.FullName = ".NET Initializer";
             app.Description = "Initializes empty project for .NET Platform";
             app.HelpOption("-h|--help");
 
-            var dotnetInit = new Program();
-            app.OnExecute((Func<int>)dotnetInit.CreateEmptyProject);
+            var dotnetNew = new Program();
+            app.OnExecute((Func<int>)dotnetNew.CreateEmptyProject);
```
Comment (davidfowl) [inline src/Microsoft.DotNet.Tools.New/Program.cs]: Whats up with this crazy cast?

Comment (krwq) [inline src/Microsoft.DotNet.Tools.New/Program.cs]: this is Func<int> vs Func<Task<int>> ambiguity

Comment (davidfowl) [inline src/Microsoft.DotNet.Tools.New/Program.cs]:
```
app.OnExecute(() => dotnetNew.CreateEmptyProject())
```

Would be cleaner

**Comment by piotrpMSFT** [comment]
Addressed all the feedback, and added a bunch of missing commands to osx packaging script. @Sridhar-MS can you validate that one?

**Comment by brthor** [comment]
LGTM, other than the Ubuntu error.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test this please.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test OSX Debug Build please
@dotnet-bot test OSX Release Build please

**Pull request merged**